indexers/common.rb

```diff
@@ -174,6 +174,68 @@ def record_is_umich(r, context)
   acc.compact!
 end
 
+#changes by mrio Feb 2022
+to_field "main_author_display", extract_marc("100abcdefgjklnpqtu4:101abcdefgjklnpqtu4:110abcdefgjklnpqtu4:111abcdefgjklnpqtu4")
+to_field "main_author", extract_marc("100abcdgjkqu:101abcdgjkqu:110abcdgjkqu:111abcdgjkqu")
+
+def concat_subfields(subfields, f)
+  subfields.split('').map{|x| f[x]}.compact.join(" ")
+end
+to_field "contributors_display" do |rec, acc|
+  ['700','710','711'].each do | tag |
+    rec.each_by_tag([tag]) do |f|
+      if f.indicator2 != "2" && f["t"].nil?
+        acc << concat_subfields('abcdefgjklnpqu4', f)
+      end
+    end
+  end
+end
+to_field "contributors" do |rec, acc|
+  subfields = 'abcdgjkqu'.split('')
+  ['700','710','711'].each do | tag |
+    rec.each_by_tag([tag]) do |f|
+      if f.indicator2 != "2" && f["t"].nil?
+        acc << concat_subfields('abcdgjkqu', f)
+      end
+    end
+  end
+end
+
+to_field "related_title_display" do |rec, acc|
+  rec.each_by_tag(["730"]) do |f|
+    if f.indicator2 == " "
+      acc << concat_subfields("abcdefghjklmnopqrstuvwxyz",f)
+    end
+  end
+  ["700","710","711"].each do |tag|
```

Comment:
Are they sure they want these 7xx personal/corporate names in the title display? Just making sure it's not a cut-and-paste error.

Reply:
This is all from this doc https://docs.google.com/spreadsheets/d/1Ic_S3LrzNy5gwpXX8VDA2GoTJ44ELt2mGJamO_LsBvM/edit#gid=0

I'm fairly certain that this is what's wanted.